README.md

```diff
@@ -20,6 +22,12 @@ and return a JSON mapping of the data to be stored in Cloudfront for a default o
 
 In the event of a cache hit, Cloudfront will return the cached value without triggering an API gateway event.
 
+### Query String Parameters
+
+| Query String Parameter | Valid Routes | Description |
+| --- | --- | --- |
+| limit | /tags | Return no more than `limit` valid tags. Must be an integer. |
```

Comment:
integers can be negative values.

Reply:
oh, good point, I'll specify that it needs to be greater than zero